app/views/site/index.html.erb

```diff
@@ -15,3 +15,46 @@
     <%= render partial: '/site/search/search_genes' %>
   </div>
 </div>
+
+<script type="text/javascript" nonce="<%= content_security_policy_script_nonce %>">
+    $(window).scroll(bindScroll);
+
+    var totalPages = <%= @studies.total_pages %>;
+    var currentPage = "<%= params[:page] %>";
+    var order = "<%= params[:order] %>";
+    var search = "<%= params[:search_terms] %>";
+    var project = "<%= params[:scpbr] %>";
+    var activeTab = 'search-studies';
+
+    $('.home-nav-tab').on('shown.bs.tab', function(el) {
+        activeTab = el.target.toString().split('#')[1];
+        console.log('active-tab: ' + activeTab );
+    });
+
+    function bindScroll(){
+        if($(window).scrollTop() + $(window).height() > $(document).height() - 1) {
+            if ( activeTab === 'search-studies') {
+                $(window).unbind('scroll');
+                paginateStudies(totalPages, order, search, project);
+            } else {
+                // event used to submit global gene search form, not attached until search is performed
+                $(window).trigger('paginateGeneSearch');
+            }
+        }
+    }
+
+    if ( currentPage == "" && totalPages == 1) {
+        $(window).unbind('scroll');
+        $("#pagination").toggle();
+    }
+
+    $("#pagination").click(function(e) {
+        e.preventDefault();
+        $(window).unbind('scroll');
+        paginateStudies(totalPages, order, search, project);
+    });
+
+    if ( search != "" ) {
```

Comment:
Two nits: we should use single- or double-quotes consistently in our JavaScript, and use strict comparison operators.  (I'd opt for using single-quotes everywhere in JS.)